templates/jvb/deployment.yaml

```
@@ -54,6 +55,18 @@ spec:
                 fieldPath: status.hostIP
             {{- end }}
           {{- end }}
+          {{- if .Values.jvb.websockets.enabled }}
+          - name: JVB_WS_SERVER_ID
+            {{- if eq $serverID "service" }}
+            value: {{ include "jitsi-meet.jvb.fullname" . }}.{{ .Release.Namespace }}.svc
+            {{- else if not (eq $serverID "podIP") }}
+            value: {{ $serverID | quote }}
+            {{- else }}
+            valueFrom:
+              fieldRef:
+                fieldPath: status.podIP
+            {{- end }}
```

```suggestion
            {{- else if eq $serverID "podIP" }}
            valueFrom:
              fieldRef:
                fieldPath: status.podIP
            {{- else }}
            value: {{ $serverID | quote }}
            {{- end }}
```

The other way around is more readable because then you are using the same logic for both if conditions.

---

Makes sense. Fixed! :)